kebechet/managers/update/update.py

```diff
@@ -797,16 +805,21 @@ def run(self, labels: list) -> Optional[dict]:
             )
 
             for e in runtime_environments:
-                self.runtime_environment = e
+                self.runtime_environment = e or "default"
                 close_no_management_issue = partial(
                     self.close_issue_and_comment,
-                    _ISSUE_NO_DEPENDENCY_NAME.format(env_name=e),
+                    _ISSUE_NO_DEPENDENCY_NAME.format(env_name=self.runtime_environment),
                     comment=ISSUE_CLOSE_COMMENT.format(sha=self.sha),
                 )
-                with cwd(thoth_config.get_overlays_directory(e)):
+                env_dir = thoth_config.get_overlays_directory(e) if e else "."
```

Comment:
is the `e` here intentional ? should this be `self.runtime_environment`

Reply:
This is on purpose, self.runtime_environment will always be set now but e still won't be.